syntax/javascript.vim

```diff
@@ -337,7 +337,7 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsSwitchColon          Noise
   HiLink jsClassMethodType      Type
   HiLink jsObjectMethodType     Type
-  HiLink jsClassDefinition      jsFuncName
+  HiLink jsClassDefinition      Constant
```

Comment:
Did you read my comment in the commit for that?  I mentioned that, yes, technically it's a function, but that users who use this style are trying to gloss over that fact.  Otherwise they'd build the thing in a prototypical way.